src/class_singleWorker.py

```diff
@@ -312,8 +315,9 @@ def sendOutOrStoreMyV3Pubkey(self, adressHash):
         """
         try:
             myAddress = shared.myAddressesByHash[adressHash]
-        except:
+        except KeyError:  # noqa:E722
             # The address has been deleted.
+            self.logger.warning("didn't work")
```

Comment:
`"Can't find %s in myAddressByHash", hexlify(addressHash)"`
I think you need hexlify, try to find out.

Reply:
Updated

Reply:
yes, hexlify will better represent addressHash